libplayerone/99-player_one_astronomy.rules

```diff
@@ -0,0 +1,8 @@
+#https://www.flir.com/support-center/iis/machine-vision/application-note/understanding-usbfs-on-linux/
+
+ACTION=="add", ATTRS{idVendor}=="a0a0", RUN+="/bin/sh -c '/bin/echo 256 >/sys/module/usbcore/parameters/usbfs_memory_mb'"
```

Comment:
This line is no longer required, it's taken care of by libindi package.

Reply:
Thank you very much for your kind assistance.
I'm very new but I want to use Playerone camera with KStars and Ekos (especially for MacOS and Raspberry Pi).
This rule was provided by SDK, however I fixed it like ASI camera.